test/integration/tests/psm.ts

```diff
@@ -458,7 +465,7 @@ describe('e2e-peg-stability-module', function () {
       });
     });
 
-    describe('mint', function () {
+    describe.skip('mint', function () {
```

Comment:
Why skip this? Because the amount of RAI on the PSM is too small? We could reduce the mint amount?

Reply:
Or maybe just remove this test. We'll be deprecating the RAI PSM as part of my empty PCV deposit PR

Reply:
we don't intend to turning minting on in the psm, no need for an e2e for it, the contract is getting deprecated in this PR